ykrt/src/compile/jitc_yk/mod.rs

```diff
@@ -71,6 +72,14 @@ impl Compiler for JITCYk {
             ));
         }
 
+        // FIXME: This should be enabled when optimisations can cope with all the inputs we see.
+        // let jit_mod = opt::opt(jit_mod)?;
+        // if should_log_ir(IRPhase::PostOpt) {
+        //     log_ir(&format!(
+        //         "--- Begin jit-post-opt ---\n{jit_mod}\n--- End jit-pre-opt ---",
```

Comment:
Random thought: later we probably will want the ability to print (and diff) individual passes at a more fine-grained level.

E.g.
```
--- Begin jit-opt-in-progress (after const_prop) ---
```

Not actionable now, obviously.

Reply:
Hmm, maybe. I think this should be a very internal detail, though, so it may not actually be worth doing in this way. Time will tell I suppose!